src/main/kotlin/org/rust/cargo/runconfig/command/CargoCommandConfiguration.kt

```diff
@@ -130,6 +129,10 @@ open class CargoCommandConfiguration(
 
     @Throws(RuntimeConfigurationError::class)
     override fun checkConfiguration() {
+        if (isRedirectInput && redirectInputPath?.toPath()?.exists() != true) {
+            throw RuntimeConfigurationWarning("Input file doesn't exist")
```

Comment:
BTW `Throws` annotation is wrong now. `checkConfiguration` can throw `RuntimeConfigurationWarning` as well, not only `RuntimeConfigurationError`
Let's use `@Throws(RuntimeConfigurationException::class)` instead

Reply:
Fixed.